NitroxInstallerActions/Patches/NitroxServerSetup.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace InstallerActions.Patches
+{
+    public static class NitroxServerSetup
+    {
+        private static string assemblyCSharpDLL = "Assembly-CSharp.dll";
```

Comment:
Would recommend refactoring:

`
private static List<string> assemblies = new List<String> {"Assembly-CSharp.dll", "Newtonsoft.Json.dll", "Assembly-CSharp-firstpass.dll"}
`

You could even do a list directory and copy all DLLs so this doesn't have to be hard coded.

Reply:
Since only specific files are used for the server I figured it would be better to not clog the server with unneeded files